### PR TITLE
openvino 2024.2.0

### PR DESCRIPTION
**Pull request by artanokhov**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/f/ffmpeg.rb [hunk 1]
```diff
@@ -6,6 +6,7 @@ class Ffmpeg < Formula
   # None of these parts are used by default, you have to explicitly pass `--enable-gpl`
   # to configure to activate them. In this case, FFmpeg's license changes to GPL v2+.
   license "GPL-2.0-or-later"
+  revision 1
```
Comment (bevanjkay) [inline Formula/f/ffmpeg.rb:9]: `ffmpeg` no longer depends on `openvino` after https://github.com/Homebrew/homebrew-core/pull/171740

Comment (artanokhov) [reply Formula/f/ffmpeg.rb:9]: Ok, thank you

**Comment by artanokhov** [comment]
@ZhongRuoyu @chenrui333
Could you please add necessary labels to extend CI timeout?

**Comment by artanokhov** [comment]
@bevanjkay 
It seems like '[CI-long-timeout](https://github.com/Homebrew/homebrew-core/labels/CI-long-timeout)' should be added
Like we have it here
https://github.com/Homebrew/homebrew-core/pull/169752

**Comment by artanokhov** [comment]
> @bevanjkay It seems like '[CI-long-timeout](https://github.com/Homebrew/homebrew-core/labels/CI-long-timeout)' should be added Like we have it here #169752

@p-linnane could you please help with that?

**Comment by chenrui333** [comment]
> It seems like '[CI-long-timeout](https://github.com/Homebrew/homebrew-core/labels/CI-long-timeout)' should be added

we dont need that label anymore, right now `long build` is good enough

**Comment by p-linnane** [comment]
We should update the text of that label to clarify.

**Comment by ZhongRuoyu** [comment]
Changed the description of https://github.com/Homebrew/homebrew-core/labels/long%20build from "Needs CI-long-timeout" to "Set a long timeout for formula testing".

**Comment by artanokhov** [comment]
Folks, there are issues with downloading manifest for ffmpeg@6
` ==> Downloading https://ghcr.io/v2/homebrew/core/ffmpeg/6/manifests/6.1.1_2
  curl: (22) The requested URL returned error: 404`
  could you please take a look?

**Comment by artanokhov** [comment]
> Folks, there are issues with downloading manifest for ffmpeg@6 ` ==> Downloading https://ghcr.io/v2/homebrew/core/ffmpeg/6/manifests/6.1.1_2 curl: (22) The requested URL returned error: 404` could you please take a look?


@chenrui333 @p-linnane @ZhongRuoyu

**Comment by bevanjkay** [comment]
@artanokhov Please be patient, our maintainer team is distributed throughout the world, but for many it is the middle of the night, someone will get to the PR when they can.
The issue here was hopefully just a temporary network issue, so I have restarted the jobs.

**Comment by artanokhov** [comment]
> @artanokhov Please be patient, our maintainer team is distributed throughout the world, but for many it is the middle of the night, someone will get to the PR when they can. The issue here was hopefully just a temporary network issue, so I have restarted the jobs.

Thank you! Really appreciate that!

**Comment by artanokhov** [comment]
We still observe error with downloading manifests
 `  ==> Downloading https://ghcr.io/v2/homebrew/core/opencv/manifests/4.9.0_10
  Error: Couldn't find manifest matching bottle checksum.
  Error: Couldn't find manifest matching bottle checksum.`

**Comment by artanokhov** [comment]
Can someone please re-run failed jobs?

**Comment by artanokhov** [comment]
As I can see currently is errors not corresponding to our libs(with quazip):
```
  /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor/portable-ruby/3.3.3/bin/bundle clean
  ==> Testing quazip
  ==> /home/linuxbrew/.linuxbrew/opt/qt/bin/qmake test.pro
  Info: creating stash file /tmp/quazip-test-20240619-258766-rs5a2f/.qmake.stash
  ==> make
  g++ -c -pipe -O2 -Wall -Wextra -fPIC -D_REENTRANT -DQT_NO_DEBUG -DQT_GUI_LIB -DQT_CORE_LIB -I. -I/home/linuxbrew/.linuxbrew/Cellar/quazip/1.4/include -I/home/linuxbrew/.linuxbrew/Cellar/qt/6.7.0_1/include -I/home/linuxbrew/.linuxbrew/Cellar/qt/6.7.0_1/include/QtGui -I/home/linuxbrew/.linuxbrew/Cellar/qt/6.7.0_1/include/QtCore -I. -I/home/linuxbrew/.linuxbrew/Cellar/qt/6.7.0_1/share/qt/mkspecs/linux-g++ -o test.o test.cpp
  In file included from /home/linuxbrew/.linuxbrew/Cellar/quazip/1.4/include/quazip/quazip.h:32,
                   from test.cpp:1:
  /home/linuxbrew/.linuxbrew/Cellar/quazip/1.4/include/quazip/zip.h:56:10: fatal error: zlib.h: No such file or directory
     56 | #include <zlib.h>
        |          ^~~~~~~~
  compilation terminated.
  make: *** [Makefile:1289: test.o] Error 1
  ==> Testing quazip (again)
  ==> /home/linuxbrew/.linuxbrew/opt/qt/bin/qmake test.pro
  Info: creating stash file /tmp/quazip-test-20240619-259002-h14grd/.qmake.stash
  ==> make
  g++ -c -pipe -O2 -Wall -Wextra -fPIC -D_REENTRANT -DQT_NO_DEBUG -DQT_GUI_LIB -DQT_CORE_LIB -I. -I/home/linuxbrew/.linuxbrew/Cellar/quazip/1.4/include -I/home/linuxbrew/.linuxbrew/Cellar/qt/6.7.0_1/include -I/home/linuxbrew/.linuxbrew/Cellar/qt/6.7.0_1/include/QtGui -I/home/linuxbrew/.linuxbrew/Cellar/qt/6.7.0_1/include/QtCore -I. -I/home/linuxbrew/.linuxbrew/Cellar/qt/6.7.0_1/share/qt/mkspecs/linux-g++ -o test.o test.cpp
  In file included from /home/linuxbrew/.linuxbrew/Cellar/quazip/1.4/include/quazip/quazip.h:32,
                   from test.cpp:1:
  /home/linuxbrew/.linuxbrew/Cellar/quazip/1.4/include/quazip/zip.h:56:10: fatal error: zlib.h: No such file or directory
     56 | #include <zlib.h>
        |          ^~~~~~~~
  compilation terminated.
  make: *** [Makefile:1289: test.o] Error 1
  Error: quazip: failed
  Error: quazip: failed
```

**Comment by artanokhov** [comment]
> As I can see currently is errors not corresponding to our libs(with quazip):
> 
> ```
>   /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor/portable-ruby/3.3.3/bin/bundle clean
>   ==> Testing quazip
>   ==> /home/linuxbrew/.linuxbrew/opt/qt/bin/qmake test.pro
>   Info: creating stash file /tmp/quazip-test-20240619-258766-rs5a2f/.qmake.stash
>   ==> make
>   g++ -c -pipe -O2 -Wall -Wextra -fPIC -D_REENTRANT -DQT_NO_DEBUG -DQT_GUI_LIB -DQT_CORE_LIB -I. -I/home/linuxbrew/.linuxbrew/Cellar/quazip/1.4/include -I/home/linuxbrew/.linuxbrew/Cellar/qt/6.7.0_1/include -I/home/linuxbrew/.linuxbrew/Cellar/qt/6.7.0_1/include/QtGui -I/home/linuxbrew/.linuxbrew/Cellar/qt/6.7.0_1/include/QtCore -I. -I/home/linuxbrew/.linuxbrew/Cellar/qt/6.7.0_1/share/qt/mkspecs/linux-g++ -o test.o test.cpp
>   In file included from /home/linuxbrew/.linuxbrew/Cellar/quazip/1.4/include/quazip/quazip.h:32,
>                    from test.cpp:1:
>   /home/linuxbrew/.linuxbrew/Cellar/quazip/1.4/include/quazip/zip.h:56:10: fatal error: zlib.h: No such file or directory
>      56 | #include <zlib.h>
>         |          ^~~~~~~~
>   compilation terminated.
>   make: *** [Makefile:1289: test.o] Error 1
>   ==> Testing quazip (again)
>   ==> /home/linuxbrew/.linuxbrew/opt/qt/bin/qmake test.pro
>   Info: creating stash file /tmp/quazip-test-20240619-259002-h14grd/.qmake.stash
>   ==> make
>   g++ -c -pipe -O2 -Wall -Wextra -fPIC -D_REENTRANT -DQT_NO_DEBUG -DQT_GUI_LIB -DQT_CORE_LIB -I. -I/home/linuxbrew/.linuxbrew/Cellar/quazip/1.4/include -I/home/linuxbrew/.linuxbrew/Cellar/qt/6.7.0_1/include -I/home/linuxbrew/.linuxbrew/Cellar/qt/6.7.0_1/include/QtGui -I/home/linuxbrew/.linuxbrew/Cellar/qt/6.7.0_1/include/QtCore -I. -I/home/linuxbrew/.linuxbrew/Cellar/qt/6.7.0_1/share/qt/mkspecs/linux-g++ -o test.o test.cpp
>   In file included from /home/linuxbrew/.linuxbrew/Cellar/quazip/1.4/include/quazip/quazip.h:32,
>                    from test.cpp:1:
>   /home/linuxbrew/.linuxbrew/Cellar/quazip/1.4/include/quazip/zip.h:56:10: fatal error: zlib.h: No such file or directory
>      56 | #include <zlib.h>
>         |          ^~~~~~~~
>   compilation terminated.
>   make: *** [Makefile:1289: test.o] Error 1
>   Error: quazip: failed
>   Error: quazip: failed
> ```

@bevanjkay could you please take a look?

**Review by bevanjkay — APPROVED** [review]

**Comment by branchvincent** [comment]
Looks like there was a merge conflict, but bottles were already published... @p-linnane do you know how to fix this?

**Comment by p-linnane** [comment]
Verified that this bottle commit correctly fetches all of the uploaded bottles (`brew fetch --bottle-tag monterey opencv`, etc.) and rebased.

**Review by p-linnane — APPROVED** [review]

**Comment by branchvincent** [comment]
Thanks @p-linnane! I’m not near my computer but I think we also need to rev bump opencv

**Pull request merged**